vql/tools/s3_upload.go

```diff
@@ -27,7 +27,7 @@ type S3UploadArgs struct {
 	CredentialsKey    string `vfilter:"required,field=credentialskey,doc=The AWS key credentials to use"`
 	CredentialsSecret string `vfilter:"required,field=credentialssecret,doc=The AWS secret credentials to use"`
 	Endpoint          string `vfilter:"optional,field=endpoint,doc=The Endpoint to use"`
-	NoVerifyCert      bool   `vfilter:"optional,field=noverifycert,doc=Skip TLS Verification"`
+	NoVerifyCert      string `vfilter:"optional,field=noverifycert,doc=Skip TLS Verification"`
```

Comment:
You should leave this as a bool.

Normally we address this in the VQL as `noverifycert=NoVerifyCert="Y"`